repository/src/main/resources/alfresco/renditions/0100-baseRenditions.json

```diff
@@ -21,7 +23,9 @@
         {"name": "allowEnlargement", "value": false},
         {"name": "maintainAspectRatio", "value": true},
         {"name": "autoOrient", "value": true},
-        {"name": "thumbnail", "value": true}
+        {"name": "thumbnail", "value": true},
+        {"name": "startPage", "value": "0"},
+        {"name": "endPage", "value": "0"}
```

Comment:
JFYI: as an aside, this may also fix https://alfresco.atlassian.net/browse/REPO-5223 (TBC via testing).